style_editor.html

```diff
@@ -99,13 +99,12 @@
 }
 
 /* Variant and Alt adjusters */
-
 .variant-alt-container {
   background: #6ADA;
   border: 1px solid black;
   font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
-  font-size: 15px;
-  padding: 2px;
+  font-size: .9em;
```

Comment:
It seems weird to define a font size in em (which is the font size)

Reply:
yeah. changed to percentage.